.github/scripts/validate_docs.py

```
@@ -0,0 +1,181 @@
+
+    # Count pipelines
+    pipeline_dirs = glob.glob("templates/pipelines/*/")
+    stats["pipeline_count"] = len(pipeline_dirs)
+
+    return stats
+
+
+def extract_readme_claims(readme_path: str) -> str:
+    """Extract relevant sections from README for validation."""
+    try:
+        with open(readme_path, "r", encoding="utf-8") as f:
```

Ruff (0.14.14)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)